src/controllers/RepostController.ts

```diff
@@ -0,0 +1,142 @@
+import { BaseController } from "./BaseController";
+import Job from "../core/Job";
+import JobPost from "../core/JobPost";
+import { JobBoard, JobInfo, PostInfo } from "../core/types";
+import { UserError } from "../utils/processUtils";
+import {
+    getJobInteractive,
+    deleteSpecificPostInteractive,
+    getAllJobPostsInteractive,
+} from "../utils/prompts";
+import { joinURL } from "../utils/pageUtils";
+import { Command } from "commander";
+import { green } from "colors";
+import Puppeteer from "puppeteer";
+
+export class RepostController extends BaseController {
+    private isInteractive: boolean;
+    private jobPostID: number;
+
+    constructor(command: Command, jobPostId: number, options: any) {
+        super(command);
+
+        this.jobPostID = jobPostId;
+        this.isInteractive = options.interactive;
+    }
+
+    async run(): Promise<void> {
+        const { browser, page } = await this.getPuppeteer();
+        await this.auth.authenticate(page);
+
+        const job = new Job(page, this.spinner, this.config);
+        let jobID;
+        let jobInfo: JobInfo;
+        const postID = this.jobPostID;
+
+        if (this.isInteractive) {
+            const { name, id } = await getJobInteractive(
+                job,
+                "What job would you like to create job posts for?",
```

Comment:
Maybe we should introduce the word "repost" somewhere, to make it different from the replicate command

Reply:
Agree. Thanks.